Mathlib/Topology/Order/LowerUpperTopology.lean

```diff
@@ -68,7 +68,7 @@ def upper (α : Type*) [Preorder α] : TopologicalSpace α := generateFrom {s |
 /-- Type synonym for a preorder equipped with the lower set topology. -/
 def WithLower (α : Type*) := α
 
-variable {α β}
+variable {α}
```

Comment:
Something quite strange. If you check out the previous commit, `f6ff4f6488c932119bf8b80c48d5d4ea94767eb7` then `exact?` was offering `exact False.elim β` as the solution to nearly everything, and `aesop` was claiming to be able to solve goals where `aesop?` would return something nonsensical.

Reply:
Ah, that's because you didn't give the type of `β` explicitly, so Lean was free to instantiate `β := False` wherever necessary!
```suggestion
variable {α β : Type*}
```
should fix it

Reply:
Ah, I did wonder why they weren't typed. Thanks.